src/shapes/Group.ts

```diff
@@ -454,7 +454,7 @@ export class Group
    * @return {Boolean}
    */
   isOnACache(): boolean {
-    return this.ownCaching || (!!this.group && this.group.isOnACache());
+    return this.ownCaching || (!!this.parent && this.parent.isOnACache());
```

Comment:
most of the time parent is unset, this would breaking the detection of isOnAcache for the objects that are not part of a multi selection.
This maybe should do something like

`const container = this.parent || this.group`

And then check the isOnACache of container?

Reply:
Ok because ActiveSelection has its own specific exitGroup.
The division between enterGroup, exitGroup, _enterGroup and _exitGroup are confusing at best.

Reply:
True